src/renderer/context.rs

```diff
@@ -0,0 +1,150 @@
+use context::get_json_pointer;
+use serde_json::value::Value;
+
+// --- module struct definitions ---
```

Comment:
sorry I meant to remove these comments in every files

Reply:
Ok, removed those.